src/Sushi.php

```diff
@@ -67,9 +61,10 @@ protected static function setSqliteConnection($database)
 
     public function migrate()
     {
-        throw_unless(is_array($this->rows), new \Exception('Sushi: $rows property not found on model: '.get_class($this)));
+        $rows = $this->getRows();
+
+        throw_unless(is_array($rows), new \Exception('Sushi: $rows property not found on model: '.get_class($this)));
```

Comment:
The exception message will be misleading when the developper overrides the `getRows()` method and doesn't return an array, i think it should be updated if you are going to keep the check there or maybe just move the check and the exception to the default `getRows()`